hydrolib/core/dflowfm/research/models.py

```diff
@@ -515,6 +558,10 @@ class Comments(Physics.Comments):
             "Uniform ground layer friction coefficient for ocean models (m/s) (0: no friction).",
             alias="uniffrictcoef1dgrlay",
         )
+        research_effectspiral: Optional[str] = Field(
+            "Deprecated keyword.",
+            alias="effectspiral",
+        )
 
```

Comment:
This was considered obsolete and to not be implemented by @arthurvd, more info in https://github.com/Deltares/HYDROLIB-core/issues/648#issuecomment-2208515383